packages/nx-plugin/project.json

```diff
@@ -9,11 +9,11 @@
       "executor": "@nx/js:tsc",
       "outputs": ["{options.outputPath}"],
       "options": {
-        "outputPath": "dist/packages/nx-plugin",
-        "main": "packages/nx-plugin/src/index.ts",
-        "tsConfig": "packages/nx-plugin/tsconfig.lib.json",
+        "outputPath": "{projectRoot}/dist",
+        "main": "{projectRoot}/src/index.ts",
+        "tsConfig": "{projectRoot}/tsconfig.lib.json",
         "assets": [
-          "packages/nx-plugin/*.md",
+          "{projectRoot}/*.md",
           {
             "input": "./packages/nx-plugin/src",
             "glob": "**/!(*.ts)",
```

Comment:
I guess only `options.assets` need to be added here? The rest is the same as in `nx.json`.

Reply:
true!

Reply:
Done!